release-please-config.json

```diff
@@ -0,0 +1,16 @@
+{
+  "packages": {
+    ".": {
```

Comment:
Should there be a package per service?

Reply:
No, otherwise we'll end up with different versions